repositories/src/main/scala/cromwell/pipeline/datastorage/dao/repository/ProjectConfigurationRepository.scala

```diff
@@ -17,6 +17,6 @@ class ProjectConfigurationRepository(repository: DocumentRepository)(implicit ec
   def updateConfiguration(projectConfiguration: ProjectConfiguration): Future[Unit] =
     upsertConfiguration(projectConfiguration)
 
-  def getById(projectId: ProjectId): Future[Option[ProjectConfiguration]] =
+  def getConfigurationById(projectId: ProjectId): Future[Option[ProjectConfiguration]] =
```

Comment:
Don't need to rename to `getConfigurationById` because it's already in `ProjectConfigurationRepository`, we can't get anything else from it

Reply:
It's my fault. Maybe when I changed Service I change it by mistake. I will try to fix it.